libraries/bot-connector/src/main/java/com/microsoft/bot/connector/authentication/SimpleCredentialProvider.java

```diff
@@ -0,0 +1,48 @@
+package com.microsoft.bot.connector.authentication;
+
+import org.apache.commons.lang3.StringUtils;
+import java.util.concurrent.CompletableFuture;
+
+public class SimpleCredentialProvider implements CredentialProvider {
+    private String _appId;
+    private String _password;
+
+
+    public SimpleCredentialProvider() {
+    }
+    public SimpleCredentialProvider(String appId, String password) {
+        this._appId = appId;
+        this._password = password;
+    }
+
+    public String getAppId() {
+        return _appId;
```

Comment:
what is the convention around "this" and "_" using both seems a little excessive?
presumably we can drop the use of "this"
(realize the C# is inconsistent but we'll be fixing that)